docs/secure-connections/sauce-connect/setup-configuration/docker.md

```diff
@@ -60,55 +92,55 @@ Docker container exposes Sauce Connect Proxy HTTP status server on port 8032. Th
 You can leverage the readiness endpoint in your CI/CD pipeline by running the following:
 
 1. Create a bash script `wait-for-sc.sh` that will ensure the pipeline only continues after Sauce Connect Proxy is fully connected and ready.
+
    ```bash title="wait-for-sc.sh"
    until [ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:8032/readiness)" == "200" ]
    do
        sleep 2
    done
    echo "SC ready"
    ```
-1. Run Sauce Connect Docker container using the script below. It is important that you map port 8032 so that the port is available to the host.
-  ```bash
-  $ docker run \
-      --detach \
-      -e SAUCE_USERNAME=${SAUCE_USERNAME} \
-      -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
-      -p 8032:8032 \
-      -t saucelabs/sauce-connect:latest \
-      -i some-identifier
-    $ ./wait-for-sc.sh
-  ```
+
+1. Run Sauce Connect Docker container using the script below. It is important that you map port 8032 (that is, `-p 8032:8032`) so that the port is available to the host.
+
+```bash
+$ docker run \
+    --detach \
+    -e SAUCE_USERNAME=${SAUCE_USERNAME} \
+    -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
+    -p 8032:8032 \
+    -t saucelabs/sauce-connect:latest \
+    -i your-docker-tunnel
+  $ ./wait-for-sc.sh
+```
 
 ### Ready File
+
 You can leverage the Sauce Connect Proxy [`--readyfile`](/dev/cli/sauce-connect-proxy/#--readyfile) flag to specify a file that will be created (or updated) when the proxy is ready.
 
 1. Create a bash script `wait-for-sc.sh` that will ensure the pipeline only continues after Sauce Connect Proxy is fully connected and ready.
-  ```bash title="wait-for-sc.sh"
-  until [ -f /tmp/sc.ready ]
-  do
-      sleep 1
-  done
-  echo "SC ready"
-  ```
-1. Run Sauce Connect Docker container using the script below. It is important that you mount a temp folder here so that `wait-for-sc.sh` can detect when Sauce Connect has launched.
-  ```bash
-  $ docker run \
-      --detach \
-      -e SAUCE_USERNAME=${SAUCE_USERNAME} \
-      -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
-      -v /tmp:/tmp \
-      --network="host" \
-      -t saucelabs/sauce-connect:latest \
-      -f /tmp/sc.ready \
-      -i some-identifier
-    $ ./wait-for-sc.sh
-  ```
 
-:::note
+```bash title="wait-for-sc.sh"
+until [ -f /tmp/sc.ready ]
+do
+    sleep 1
+done
+echo "SC ready"
+```
 
-`--network="host"` allows Sauce Connect Proxy to access your app in the host machine. It's only required if your app runs on the same machine as the docker host.
+1. Run Sauce Connect Docker container using the script below. It is important that you mount a temp folder (that is, `-v /tmp:/tmp`) so that `wait-for-sc.sh` can detect when Sauce Connect is ready.
```

Comment:
```suggestion
1. Run Sauce Connect Docker container using the script below. You must mount a temporary folder (`-v /tmp:/tmp`) so that `wait-for-sc.sh` can detect when Sauce Connect is ready.
```